src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.nav {
+  margin-right: 50px;
+}
```

Comment:
`nav` should have margins on its left and right sides
```
.nav {
  margin: 0 50px;
}
```
doing so is the same as this one
```
.nav {
  margin-right: 50px;
  margin-left: 50px
}
```